helm-charts/common/data-prep/README.md

```diff
@@ -37,21 +87,40 @@ Then run the command `kubectl port-forward svc/data-prep 6007:6007` to expose th
 
 Open another terminal and run the following command to verify the service if working:
 
-```console
+### 1. For Data-prep service using redis-vector-db:
+
+```bash
+
 curl http://localhost:6007/v1/dataprep  \
     -X POST \
     -H "Content-Type: multipart/form-data" \
     -F "files=@./README.md"
 ```
 
+### 2. For multimodal data prep service using vdms-vector-db:
+
+```bash
+# 1) Download a sample video in current dir:
+curl -svLO "https://github.com/opea-project/GenAIExamples/raw/refs/heads/main/VideoQnA/docker_compose/intel/cpu/xeon/data/op_1_0320241830.mp4"
```

Comment:
Would it not be better for this sample video to live in some common VidoQnA directory versus in docker_compose/intel/cpu/xeon/data

Reply:
I suppose requested change needs a PR in GenAIExamples rather than here.